output/transform/merge_partition_sizes.go

```diff
@@ -0,0 +1,56 @@
+package transform
+
+import (
+	snapshot "github.com/pganalyze/collector/output/pganalyze_collector"
+	"github.com/pganalyze/collector/state"
+)
+
+func mergePartitionSizes(s snapshot.FullSnapshot, newState state.PersistedState, ts state.TransientState, databaseOidToIdx OidToIdx) snapshot.FullSnapshot {
+	for idx, info := range s.IndexInformations {
+		rel := s.RelationInformations[info.RelationIdx]
+		if !rel.HasParentRelation || rel.PartitionBoundary == "" {
+			continue
+		}
+		for parentIdx, pi := range s.IndexInformations {
+			if pi.RelationIdx != rel.ParentRelationIdx {
+				continue
+			}
+			if info.IndexType == pi.IndexType && info.IsUnique == pi.IsUnique && intArrayEqual(info.Columns, pi.Columns) {
+				s.IndexStatistics[parentIdx].SizeBytes += s.IndexStatistics[idx].SizeBytes
```

Comment:
I forgot what it was previously (when it was in the previous PR), but are we only tracking the size of Index stats?